### ISSUE TITLE
Loss value

### ISSUE THREAD

**Issue by LYKlyk** [Chen-Junbao/CPM#2]
Thank you very much for your work. I have a problem: I find that if I directly use the Heatmap output from the network to compare with the generated Heatmap, the loss value will be abnormally large. How to deal with this problem?

**Comment by Chen-Junbao** [comment]
Is your own training code like this?
```python
for j, data in enumerate(train_loader):
			inputs, heatmap, centermap = data

			inputs = inputs.cuda()
			heatmap = heatmap.cuda(async=True)
			centermap = centermap.cuda(async=True)

			input_var = torch.autograd.Variable(inputs)
			heatmap_var = torch.autograd.Variable(heatmap)
			centermap_var = torch.autograd.Variable(centermap)

			heat1, heat2, heat3, heat4, heat5, heat6 = model(input_var, centermap_var)

			loss1 = criterion(heat1, heatmap_var)
			loss2 = criterion(heat2, heatmap_var)
			loss3 = criterion(heat3, heatmap_var)
			loss4 = criterion(heat4, heatmap_var)
			loss5 = criterion(heat5, heatmap_var)
			loss6 = criterion(heat6, heatmap_var)

			loss = loss1 + loss2 + loss3 + loss4 + loss5 + loss6
```
I remember when I was training my model, the loss value was also very large at first, but after 12 hours of training it was smaller。
